stdlib/public/core/RandomAccessCollection.swift

```diff
@@ -152,7 +155,9 @@ where SubSequence: RandomAccessCollection, Indices: RandomAccessCollection
   ///   is the same value as the result of `abs(distance)` calls to
   ///   `index(before:)`.
   ///
-  /// - Complexity: O(1)
+  /// - Complexity: O(1) if the collection conforms to
+  ///   `RandomAccessCollection`; otherwise, O(*k*), where *k* is the absolute
+  ///   value of `distance`.
```

Comment:
Didn't you pr the removal of `if the collections conforms to...`?

Reply:
I did this edit before that (timeline doesn't say so, because I had trouble with Github Desktop not wanting to create a branch on my fork.
Will re-fix.